### ISSUE TITLE
Email Field doesn't allow for valid email inputs

### ISSUE THREAD

**Issue by superb-sushi** [superb-sushi/pe#1]
### Description

![image.png](https://raw.githubusercontent.com/superb-sushi/pe/main/files/8598bb26-bbfd-4add-9459-43619074699e.png)

Email addresses may accept the special characters (+_.-%) in the `local-part` section of the email (https://www.designsystem.tech.gov.sg/patterns/email-address#:~:text=Email%20Address-,Email%20Address,-Use%20this%20pattern).

### Expected
The input given should be accepted, as all fields adhere to conventions.

### Actual
Email field `e/test%test@test.com` is rejected (due to presence of `%` character).

### Severity
>  severity.Medium : A flaw that causes occasional inconvenience to some users, but they can continue to use the product.

If the user's client uses a similar email, the user would not be able to add their client to the app, resulting in inconvenience for the user.

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Symbols like “%” and “!” are indeed considered part of a valid email. However, historically they have had special meanings are operators (specifically, as routing characters to route incoming/outgoing mail between specific inboxes and UUCP gateways).

You may refer to [this thread](https://groups.google.com/g/comp.mail.headers/c/MDZPvKXTb3w) for some more details. A quote from the thread:
 
>Most addresses with percents are now lame attempts at spamming, so you won't be doing anyone any favors by accepting them.


TLDR even back in 2007 it had become very uncommon to use % in an email address

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
